coldfront/core/utils/validate.py

```diff
@@ -70,13 +60,9 @@ def __init__(self, value):
 
     def validate_billing_cycle(self):
         try:
-            validate = validators.OneOf(BILLING_CYCLE_OPTIONS)
+            validate = validators.OneOf([option[0] for option in BILLING_CYCLE_OPTIONS])
             validate.to_python(self.value)
         except:
             raise ValidationError(
-                f"Invalid Value {self.value}. Value must be one of {BILLING_CYCLE_OPTIONS}."
+                f"Invalid Value {self.value}. Value must be one of {[option[0] for option in BILLING_CYCLE_OPTIONS]}."

```

Comment:
[Minor] At the very least, it should display `option[1]` since this is the humanized version. 

[Improvement] join the string with commas.
```
', '.join([option[1] for option in BILLING_CYCLE_OPTIONS])
```

Reply:
Good catch.